internal/internal_worker_base.go

```diff
@@ -116,7 +116,7 @@ type (
 		// Application level code must be executed from this function only.
 		// Execute call as well as callbacks called from WorkflowEnvironment functions can only schedule callbacks
 		// which can be executed from OnWorkflowTaskStarted().
-		OnWorkflowTaskStarted()
+		OnWorkflowTaskStarted(timeout time.Duration)
```

Comment:
rename to deadlockDetectionTimeout

Reply:
This was actually intentional, as I think timeout makes perfect sense in this context. However I'm fine changing it.